torch/csrc/jit/codegen/cuda/fusion_segmenter.cpp

```diff
@@ -75,7 +133,7 @@ std::vector<SegmentedGroup::NeighborGroup> SegmentedGroup::
     return {};
   }
 
-  std::vector<bool> can_merge(true, neighbors.size());
+  std::vector<bool> can_merge(neighbors.size(), true);
```

Comment:
😨

Reply:
Was a tricky one to find and it made quite far...

Reply:
That's why I almost always use `at()` rather than `[]`. This type of memory violations is really annoying to debug.